### PR TITLE
pre audit fixes

### PR DESCRIPTION
**Pull request by dcbuild3r**
Fixing review comments

### CODE REVIEW

File: src/StateBridge.sol [hunk 1]
```diff
@@ -9,15 +9,22 @@ import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
...
+/// @author Worldcoin
+/// @notice Distributes new World ID Identity Manager roots to World ID supported networks
+/// @dev This contract lives on Ethereum mainnet and is called by the World ID Identity Manager contract
+/// in the registerIdentities method
 contract StateBridge is FxBaseRootTunnel, Ownable {
-    /// @notice boilerplate property to satisfy FxBaseRootTunnel inheritance (not going to be used)
-    bytes public latestData;
+    /*//////////////////////////////////////////////////////////////
+                                STORAGE
+    //////////////////////////////////////////////////////////////*/
```
Comment (iamrecursion) [inline src/StateBridge.sol:22]: Please use the separator style that the identity manager does. Better to be consistent!

File: src/StateBridge.sol [hunk 2]
```diff
@@ -45,15 +56,23 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
     /// @param timestamp The Ethereum block timestamp of the latest WorldID Identity Manager root.
     event RootSentToPolygon(uint256 root, uint128 timestamp);
 
-    /// @notice Emmited when the root is not a valid root in the canonical WorldID Identity Manager contract
-    error InvalidRoot();
+    /*//////////////////////////////////////////////////////////////
+                                 ERRORS
+    //////////////////////////////////////////////////////////////*/
+
+    /// @notice Thrown when caller of sendRootMultichain is not the WorldID Identity Manager contract
```
Comment (iamrecursion) [inline src/StateBridge.sol:63]:
```suggestion
    /// @notice Thrown when the caller of `sendRootMultichain` is not the WorldID Identity Manager contract.
```

File: src/StateBridge.sol [hunk 3]
```diff
@@ -132,8 +160,12 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
     /// @param root The latest WorldID Identity Manager root to be sent to Polygon
     /// @param timestamp The Ethereum block timestamp of the latest WorldID Identity Manager root
     function _sendRootToPolygon(uint256 root, uint128 timestamp) internal {
+        console.log("sendRootToPolygon called");
```
Comment (iamrecursion) [inline src/StateBridge.sol:163]: Shouldn't be here.

File: src/StateBridge.sol [hunk 4]
```diff
@@ -9,15 +9,22 @@ import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
 import {IWorldIDIdentityManager} from "./interfaces/IWorldIDIdentityManager.sol";
 import {ICrossDomainOwnable3} from "./interfaces/ICrossDomainOwnable3.sol";
 import {FxBaseRootTunnel} from "fx-portal/contracts/tunnel/FxBaseRootTunnel.sol";
+import "forge-std/console.sol";
```
Comment (iamrecursion) [inline src/StateBridge.sol:12]: Remove this.

File: src/StateBridge.sol [hunk 5]
```diff
@@ -91,15 +116,16 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
     /// @param root The latest WorldID Identity Manager root.
     /// @param timestamp The Ethereum block timestamp of the latest WorldID Identity Manager root.
     function _sendRootToOptimism(uint256 root, uint128 timestamp) internal {
-        bytes memory message;
-
-        message = abi.encodeCall(IOpWorldID.receiveRoot, (root, timestamp));
+        console.log("sendRootToOptimism called");
```
Comment (iamrecursion) [inline src/StateBridge.sol:119]: Remove this.

**Review by iamrecursion — APPROVED** [review]

File: src/StateBridge.sol (hunk 1 above)
Comment (dcbuild3r) [reply src/StateBridge.sol:22]: I have a rust binary that I call from within VSCode that does this automatically for any word I type. What tool do you use for it?

File: src/StateBridge.sol (hunk 5 above)
Comment (dcbuild3r) [reply src/StateBridge.sol:119]: done

File: src/StateBridge.sol (hunk 4 above)
Comment (dcbuild3r) [reply src/StateBridge.sol:12]: done

File: src/StateBridge.sol (hunk 3 above)
Comment (dcbuild3r) [reply src/StateBridge.sol:163]: done

**Pull request merged**